spec/std/dir_spec.cr

```diff
@@ -257,6 +257,14 @@ describe "Dir" do
   it "lists entries" do
     filenames = Dir.entries(__DIR__)
     filenames.includes?("dir_spec.cr").should be_true
+    filenames.includes?(".").should be_true
+    filenames.includes?("..").should be_true
+  end
+
+  it "lists children" do
+    filenames = Dir.children(__DIR__)
+    filenames.includes?(".").should be_false
+    filenames.includes?("..").should be_false
```

Comment:
How about comparisons comparing the output to `Dir.entries` here? Like checking that the size is the same minus two and that all returned items are also in the other.